### ISSUE TITLE
Format ${query.month} as a number

### ISSUE THREAD

**Issue by JayAige** [kevinpapst/kimai2#1025]
Currently the variable ${query.month} generates the month on the invoice as word (docx renderer). Is there a possibility to export this as a number? For example instead of 'July', '07 will be exported.
Thank you in advance.

**Comment by kevinpapst** [comment]
Sure there is a possibility. But not without sending a PR ;-)
Add something like:
```
'query.month.number' => $model->getQuery()->getBegin()->format('m'),
```
after:
https://github.com/kevinpapst/kimai2/blob/master/src/Invoice/Renderer/RendererTrait.php#L108

**Comment by kevinpapst** [comment]
merged to master, will be available with 1.2
